src/zc/ffscript.cpp

```diff
@@ -13353,51 +13353,11 @@ int32_t get_register(int32_t arg)
 			} 
 			else 
 			{ 
-				switch(indx)
+				if (indx < 0|| indx >= 32)
```

Comment:
Might as well use the constant if you have it
```suggestion
				if (indx < 0|| indx >= MAX_NPC_ATRIBUTES)
```